lib/tasks/limber.rake

```diff
@@ -244,7 +250,8 @@ namespace :limber do
           'Ribozero RNA depletion',
           'Ribozero RNA-seq (Bacterial)',
           'Ribozero RNA-seq (HMR)',
-          'TraDIS'
+          'TraDIS',
+          'Chromium Visium'
```

Comment:
Is Visium here because it was missed out in the previous release?

Reply:
yes